driver/api/impl/handles.cpp

```diff
@@ -9,7 +9,7 @@
 
 #include <type_traits>
 
-namespace {
+namespace { namespace impl {
```

Comment:
why do you need an additional namespace here? Aren't unnamed namespace are unique and non-visible?

Reply:
Just for readability/code organization: `impl::abc()` looks better than `impl_abc()`.